fetch.md

```diff
@@ -1,11 +1,204 @@
+ephemeral resource cache, and use their "typical" HTTP cache partition.
+
+Like the main proposal, we could potentially provide an optional cache merging step in which resources stored
+in the ephemeral cache could be transferred to the HTTP cache for a more-permanent lifetime. This is not without
+its own difficulties and complexities as previously described.
+
+#### Bypassing the HTTP cache entirely
+
+Another approach to this problem would be to force-send all requests that were initiated even indirectly
+by a [prerendering browsing context][4] with the [cache mode][3] value of `"no-store"`. Fetching would behave
+as though HTTP cache were not present, and it would be impossible to contaminate any HTTP cache partitions observable
+to other pages.
+
+The main downside of this approach is that we lose the ability to use any of the resources fetched by
+the [prerendering browsing context][4] after activation. That is, we lose whatever benefits we could
+gain by having the ability to merge speculative <=> "typical" HTTP cache partitions, should we decide
+to go that route.
+
+On a similar note, this approach does not integrate with `<link rel=prefetch>`, as the whole point is
+to cache a top-level resource somewhere it can be usefully retrieved upon navigation. This means we're likely going to have
+to spec and implement the HTTP cache partitioning mechanics described above anyways, so it is sensible for
+other speculative navigation requests to piggy-back off of the same infrastructure.
+
+Another possible complexity is that if we spec prerendering in a way such that [prerendering browsing contexts][4]
+can be thrown away under implementation-defined constraints at any time, then we lose the ability to ever profit
+from the prerendering work we've done. No resources in the [prerendering browsing contexts][4] are preserved
+elsewhere, so it would be impossible to downgrade prerendering activation to something like prefetch activation.
+
+> TODO(domfarolino): Clarify that the above is only considering the downgrading of prerender *activations*. There is
+another kind of downgrade that we may want to consider, which is the scenario where under implementation-defined
+constraints, the UA opts not to create a [prerendering browsing context][4], but instead downgrade the whole prerender
+process to a simple prefetch. None of this is set in stone though.
+
+## Fetching With No Credentials
+
+Speculative navigation requests should in no way leak the user's identity to third-party origins. A consequence of
+this is that we must fetch all requests originating from a [prerendering browsing context][4] with no credentials.
+As with the HTTP cache partitioning restrictions, this must apply to all of the following:
+ - Top-level speculative resources
+ - Subresources under top-level speculative resources
+ - Nested browsing contexts and their subresources
```

Comment:
IIRC part of the plan is to delay loading nested browsing contexts until activation. I'm unsure whether we'll do that at the fetch level (i.e., just make the fetch hang) or as part of the spec for nested browsing contexts, hmm.

I don't think we need to even mention it here, but it's something for us to keep in mind...

Reply:
Well, OK, upon further reflection, we probably want to do it as part of the spec for nested browsing contexts. Then the credentials just work, with no issues.

Reply:
OK, as per that and some offline discussion we'll leave this for now since it highlights the important constraint, and we'll figure out exactly how we achieve this constraint later through the spec design.